tests/cpp/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # Copyright (c) 2013-2015, EPFL/Blue Brain Project
 #                     Daniel Nachbaur <daniel.nachbaur@epfl.ch>
 #                     Raphael Dumusc <raphael.dumusc@epfl.ch>
+#
+# Change this number when adding tests to force a CMake run: 0
```

Comment:
Or you pull, cmake and make, and if you forgot the cmake step the test you're looking for just simply does not exists yet, so you run cmake at this point... ;-)